Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -267,50 +258,43 @@ ActsExamples::ProcessCode ActsExamples::SeedingAlgorithm::execute(
       up - m_cfg.seedFinderConfig.deltaRMiddleMaxSPRange);
 
   // run the seeding
-  static thread_local SimSeedContainer seeds;
+  static thread_local std::vector<seed_type> seeds;
   seeds.clear();
   static thread_local decltype(m_seedFinder)::SeedingState state;
-  state.spacePointData.resize(
-      spacePointPtrs.size(),
-      m_cfg.seedFinderConfig.useDetailedDoubleMeasurementInfo);
-
-  if (m_cfg.seedFinderConfig.useDetailedDoubleMeasurementInfo) {
-    for (std::size_t grid_glob_bin(0);
-         grid_glob_bin < spacePointsGrouping.grid().size(); ++grid_glob_bin) {
-      const auto& collection = spacePointsGrouping.grid().at(grid_glob_bin);
-      for (const auto& sp : collection) {
-        std::size_t index = sp->index();
-
-        const float topHalfStripLength =
-            m_cfg.seedFinderConfig.getTopHalfStripLength(sp->sp());
-        const float bottomHalfStripLength =
-            m_cfg.seedFinderConfig.getBottomHalfStripLength(sp->sp());
-        const Acts::Vector3 topStripDirection =
-            m_cfg.seedFinderConfig.getTopStripDirection(sp->sp());
-        const Acts::Vector3 bottomStripDirection =
-            m_cfg.seedFinderConfig.getBottomStripDirection(sp->sp());
-
-        state.spacePointData.setTopStripVector(
-            index, topHalfStripLength * topStripDirection);
-        state.spacePointData.setBottomStripVector(
-            index, bottomHalfStripLength * bottomStripDirection);
-        state.spacePointData.setStripCenterDistance(
-            index, m_cfg.seedFinderConfig.getStripCenterDistance(sp->sp()));
-        state.spacePointData.setTopStripCenterPosition(
-            index, m_cfg.seedFinderConfig.getTopStripCenterPosition(sp->sp()));
-      }
-    }
-  }
+  auto stop_preparation = std::chrono::high_resolution_clock::now();
 
+  auto start = std::chrono::high_resolution_clock::now();
   for (const auto [bottom, middle, top] : spacePointsGrouping) {
     m_seedFinder.createSeedsForGroup(
         m_cfg.seedFinderOptions, state, spacePointsGrouping.grid(),
         std::back_inserter(seeds), bottom, middle, top, rMiddleSPRange);
   }
+  auto stop = std::chrono::high_resolution_clock::now();
+
+  auto duration_preparation =
+      std::chrono::duration_cast<std::chrono::nanoseconds>(stop_preparation -
+                                                           start_preparation)
+          .count();
+  auto duration =
+      std::chrono::duration_cast<std::chrono::nanoseconds>(stop - start)
+          .count();
+  std::cout << "preparation time=" << duration_preparation << "\n";
+  std::cout << "seeding time=" << duration << "\n";
```

Comment:
I am assuming the couts and the chrono timers are here just for testing. 

Do you know the effects of this PR in time performance?

Reply:
yup, it's for debugging. I have not completed yet the timing studies, but I'm taking a look at them